playbooks/operationalizing/secure-forward-splunk-container.adoc

```diff
@@ -0,0 +1,289 @@
+---
+---
+= Forwarding Logs to Splunk Using an External Containerized Fluentd Secure Forward
+Karl Moos <moos@redhat.com>
+v1.0, 2018-01-31
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+Send logs captured by the OpenShift Container Platform link:https://docs.openshift.com/container-platform/latest/install_config/aggregate_logging.html[aggregated logging] solution to Splunk using the link:http://docs.fluentd.org/v0.12/articles/out_secure_forward[Fluentd secure forward output plugin]
```

Comment:
We should be sure to clarify this is the containerized solution instead of the standalone fluentd